src/extractor/restriction_parser.cpp

```diff
@@ -126,6 +126,25 @@ RestrictionParser::TryParse(const osmium::Relation &relation) const
         {
             return {};
```

Comment:
Noticed that here we do `return {}` if found something unexpected in tag values, should we just ignore it instead? may be with some additional warning

Reply:
In most cases you'll get at most one restriction returned per relation, it's only no entry/exits that could have multiple.

I think the way to view it is, if the relation is invalid, how correct will the remaining parsed restrictions be? It's difficult to know, so it's cleaner to not to try and deal with them.

We could add a log, but I would only make it a debug level. It's not OSRM's problem if input data is not correct.